shopify-client/src/views/Cart.vue

```diff
@@ -45,7 +59,9 @@
         </b-row>
         <b-row>
           <b-col>
-            <button class="btn btn-success">Checkout</button>
+            <button class="btn btn-success" v-on:click="checkout()">
```

Comment:
attributes should be split on newlines

Reply:
What do you mean?

Reply:
class
v-on

Reply:
I will setup a fromatter later for code styling.